agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/heartbeat/WebAppsHeartbeatProvider.java

```diff
@@ -52,6 +52,13 @@
 
   private static final String WEBSITE_HOME_STAMPNAME = "appSrv_wsStamp";
 
+  private static final String WEBSITE_OWNER_NAME = "appSrv_wsOwner";
+
```

Comment:
does spotlessApply remove the empty line in between instance variables?

Reply:
looks like it doesnot remove empty line.